[2023-03-29 03:46:50,659] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: SP_500_DATA_PIPELINE_v2.extract_data_task manual__2023-03-29T03:46:41.289486+00:00 [queued]>
[2023-03-29 03:46:50,867] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: SP_500_DATA_PIPELINE_v2.extract_data_task manual__2023-03-29T03:46:41.289486+00:00 [queued]>
[2023-03-29 03:46:50,890] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 03:46:50,892] {taskinstance.py:1239} INFO - Starting attempt 1 of 3
[2023-03-29 03:46:50,895] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 03:46:51,052] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): extract_data_task> on 2023-03-29 03:46:41.289486+00:00
[2023-03-29 03:46:51,068] {standard_task_runner.py:52} INFO - Started process 738 to run task
[2023-03-29 03:46:51,089] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'SP_500_DATA_PIPELINE_v2', 'extract_data_task', 'manual__2023-03-29T03:46:41.289486+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp8o1p0riz', '--error-file', '/tmp/tmps9008d7z']
[2023-03-29 03:46:51,096] {standard_task_runner.py:77} INFO - Job 62: Subtask extract_data_task
[2023-03-29 03:46:51,285] {logging_mixin.py:109} INFO - Running <TaskInstance: SP_500_DATA_PIPELINE_v2.extract_data_task manual__2023-03-29T03:46:41.289486+00:00 [running]> on host 5ebc180c7cc0
[2023-03-29 03:46:51,553] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=SP_500_DATA_PIPELINE_v2
AIRFLOW_CTX_TASK_ID=extract_data_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T03:46:41.289486+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-29T03:46:41.289486+00:00
[2023-03-29 03:47:03,181] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/pandas_datareader/tiingo.py:234: FutureWarning: In a future version of pandas all arguments of concat except for the argument 'objs' will be keyword-only
  return pd.concat(dfs, self._concat_axis)

[2023-03-29 03:47:03,508] {logging_mixin.py:109} INFO - ingestion from api completed
[2023-03-29 03:47:03,990] {logging_mixin.py:109} INFO - file has been saved at :sp_500_data.csv
[2023-03-29 03:47:03,992] {logging_mixin.py:109} INFO - sp_500_data.csv
[2023-03-29 03:47:03,994] {python.py:175} INFO - Done. Returned value was: None
[2023-03-29 03:47:04,190] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=SP_500_DATA_PIPELINE_v2, task_id=extract_data_task, execution_date=20230329T034641, start_date=20230329T034650, end_date=20230329T034704
[2023-03-29 03:47:04,688] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-03-29 03:47:05,309] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
